services/web/client/source/class/osparc/component/export/StudyPermissions.js

```diff
@@ -66,12 +66,7 @@ qx.Class.define("osparc.component.export.StudyPermissions", {
     __studyData: null,
 
     _isUserOwner: function() {
```

Comment:
do you actually still need the function?

Reply:
Nope, thanks

Reply:
Yes, it needs to override the base class (servicePermissions are checked differently)